### PR TITLE
Cleanup old maya-cli code

### PR DESCRIPTION
**Pull request by prateekpandey14**
**What this PR does / why we need it**:
This commit will cleanup the old maya-cli code from and remove its dependencies from vendor.

**Which issue this PR fixes**
Fixes openebs/openebs#1108
Fixes openebs/openebs#505

### CODE REVIEW

**Comment by prateekpandey14** [comment]
Need to fix the Makefile for the build dir `bin`

File: buildscripts/mayactl/build.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ set -e
 # Get the parent directory of where this script is.
 SOURCE="${BASH_SOURCE[0]}"
 while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
-DIR="$( cd -P "$( dirname "$SOURCE" )/../../cmd/mayactl" && pwd )"
+DIR="$( cd -P "$( dirname "$SOURCE" )/../../" && pwd )"
```
Comment (kmova) [inline buildscripts/mayactl/build.sh]: Is this the change, that is forcing us to add ./cmd/mayactl in the below lines.

**Comment by kmova** [comment]
@prateekpandey14 It will be good to add a commit as part of this PR that cleans up the vendor as well.

File: buildscripts/mayactl/build.sh (hunk 1 above)
Comment (prateekpandey14) [reply buildscripts/mayactl/build.sh]: No , above was added by me , then removed

**Comment by prateekpandey14** [comment]
There will be another PR to remove and update the vendor deps

**Pull request merged**